### DISCUSSION TITLE
PAN-OS SDK API Reference and Configuration Tree Diagrams & Module Base Broken

### DISCUSSION THREAD

**Discussion by Gitterdone8** [PaloAltoNetworks/pan-os-python#558] [General]
The PAN-OS-SDK  API reference documentation is broken and will not come up.

When going to Read The Docs for the PAN-OS-SDK and going to [](https://pan-os-python.readthedocs.io/en/latest/configtree.html) the configuration tree is broken.  This means that the examples and methods for the various modules can't be pulled up.

I tried using the Wayback Machine to find earlier instances before it was broken and the results are the same.

Whether I use the index to open a module, perform a search, etc., nothing works.  The only thing that is not broken is the getting started and related examples.

It would be very appreciated if this get get resolved as soon as possible.

Thank you

**Comment by jvictorgonc** [comment]
Yeahh, i could not verify the API reference as well. Any news about this issue?

**Comment by Gitterdone8** [comment]
This was working with the stable version.  It is now broken with both the stable and latest version.

**Comment by BatD2** [comment]
This is sitll not working even after the latest fix. The diagrams are now showing, but the actual object references are now there.

**Comment by da110us** [comment]
It's crazy... Still one mess. I think Palo Alto abandoned the Python SDK. I don't think there is any other way to figure out how to use the SDK right?

Why don't they fix it?

**Comment by da110us** [comment]
It should be fixed, but it looks like it waits for some kind of review. I see a lot of pull requests that need review. Some for quite some time. Does anybody knows how this is supposed to work? Must somebody from Palo Alto review/approve these changes?
